### PR TITLE
Add credit_specification parameter to ess scaling configuration

### PR DESCRIPTION
**Pull request by ilya-v-trofimov**

### CODE REVIEW

File: alicloud/resource_alicloud_ess_scalingconfiguration.go [hunk 1]
```diff
@@ -431,6 +439,11 @@ func modifyEssScalingConfiguration(d *schema.ResourceData, meta interface{}) err
 		d.SetPartial("internet_max_bandwidth_out")
 	}
 
+	if d.HasChange("credit_specification") {
+		request.SystemDiskCategory = d.Get("credit_specification").(string)
```
Comment (xiaozhu36) [inline alicloud/resource_alicloud_ess_scalingconfiguration.go:443]: I think it should be `request.CreditSpecification = d.Get("credit_specification").(string)`

File: website/docs/d/ess_scaling_configurations.html.markdown [hunk 2]
```diff
@@ -50,6 +50,7 @@ The following attributes are exported in addition to the arguments listed above:
   * `internet_charge_type` - Internet charge type of the scaling configuration.
   * `internet_max_bandwidth_in` - Internet max bandwidth in of the scaling configuration.
   * `internet_max_bandwidth_out` - Internet max bandwidth of the scaling configuration.
+  * `credit_specification` - Performance mode of the t5 burstable instance.
```
Comment (xiaozhu36) [inline website/docs/d/ess_scaling_configurations.html.markdown:53]: This is datasource output and if you want to support it, it should be added in its data source code.

**Review by xiaozhu36 — CHANGES_REQUESTED** [review]

File: alicloud/resource_alicloud_ess_scalingconfiguration.go (hunk 1 above)
Comment (ilya-v-trofimov) [reply alicloud/resource_alicloud_ess_scalingconfiguration.go]: Yeah, my bad. Fixed now.

File: website/docs/d/ess_scaling_configurations.html.markdown (hunk 2 above)
Comment (ilya-v-trofimov) [reply website/docs/d/ess_scaling_configurations.html.markdown:53]: Fair enough. Added to datasources now

File: website/docs/r/ess_scaling_configuration.html.markdown [hunk 3]
```diff
@@ -106,6 +106,7 @@ The following arguments are supported:
 * `internet_charge_type` - (Optional) Network billing type, Values: PayByBandwidth or PayByTraffic. Default to `PayByBandwidth`.
 * `internet_max_bandwidth_in` - (Optional) Maximum incoming bandwidth from the public network, measured in Mbps (Mega bit per second). The value range is [1,200].
 * `internet_max_bandwidth_out` - (Optional) Maximum outgoing bandwidth from the public network, measured in Mbps (Mega bit per second). The value range for PayByBandwidth is [0,100].
+* `credit_specification` - (Optional) Performance mode of the t5 burstable instance. Valid values: 'Standard', 'Unlimited'.
```
Comment (xiaozhu36) [inline website/docs/r/ess_scaling_configuration.html.markdown:109]: The next version is 1.98.0 and if you add a new resource attribute, you should show its available version, like 
```
 `credit_specification` - (Optional, Available in 1.98.0+) 
```

**Review by xiaozhu36 — CHANGES_REQUESTED** [review]

File: website/docs/r/ess_scaling_configuration.html.markdown (hunk 3 above)
Comment (ilya-v-trofimov) [reply website/docs/r/ess_scaling_configuration.html.markdown]: Done. 
Let me know if I need to add the same into 'datasource' markdown as well

**Review by xiaozhu36 — APPROVED** [review]

**Pull request merged**